src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/_scaffold.scss

```diff
@@ -17,17 +18,212 @@ main, #pagehdr, footer {
 
 }
 
+@mixin icon_label($icon, $label) {
+
+  @include flex;
+  @include flex-items-column;
+  margin-right: 1ch;
+
+  .icon {
+    align-self: center;
+    width: 32px; height: 24px;
+    // image-url() is not working...
+    background-image: url("../../resource/img/#{$icon}");
+    background-size: 32px 24px;
+    background-repeat: no-repeat;
+
+  }
+  .icon+* {
+    align-self: center;
+    text-transform: uppercase;
+    font-size: $font-size-smaller;
+  }
+}
+
 // for article
-main{
+main {
+
+  align-self: center;
+  max-width: 920px;
+  width: 100%;
+
+  .article-meta {
+    // cannot use grid as it folds to single column
+    display: flex; flex-flow: row nowrap;
+    justify-content: space-between;
+    font-size: $font-size-smaller;
+    color: $brand-main;
+    margin-top: 4ch;
+    width: 100%;
+
+
+    .peer-reviewed, .preprint-checks {
+      text-transform: uppercase;
+      text-align: center;
+      flex: 1;
+      max-width: 100px;
+    }
+  }
 
   article {
-    margin: $pad-medium $grid-gutter;
+    padding: $pad-medium $grid-gutter;
+    box-sizing: border-box;
+    border: solid 1px $color-border;
+    @include breakpoint(sm) {
+      border: none;
+    }
+
+    @include flex;
+    @include flex-items-column;
+    @include flex-items-nowrap;
+
+    .article-info-block {
+      @include flex;
+      @include flex-items-row;
+      @include flex-items-nowrap;
+      @include flex-items-space-between;
+      @include breakpoint(sm) {
+        @include flex-items-space-around;
+      }
+
+      .date-doi {
+        @include breakpoint(sm) {
+          text-align: center;
+        }
+        font-family: $headings-font-family;
+        margin-bottom: 4ch;
+
+        .date {
+          font-size: $font-size-big;
```

Comment:
It looks like the font styles are reused throughout the design. we could pull this out into a typography stylesheet.